ui/src/pages/ResultsPage.tsx

```diff
@@ -4776,6 +4776,27 @@ const InventoryTabContent = (props: {
 		return data;
 	};
 
+	const getCICDRows = (configsSeparator: string = ",") => {
+		const result = [];
+		for (const item of Object.values(
+			scan.results?.inventory?.cicd_tools ?? {},
+		)) {
+			result.push({
+				tool: item.display_name,
+				files: item.configs
+					.map((config: { [key: string]: string }) => config.path)
+					.join(configsSeparator),
+			});
+		}
+		return result;
+	};
+
+	const cicdToolsColumns: ColDef[] = [
+		{ field: "tool", headerName: i18n._(t`Tool`) },
+		{ field: "files", headerName: i18n._(t`Config Files`) },
+	];
+	const cicdToolsRows: RowDef[] = getCICDRows(", ");
```

Comment:
Looking at sample output, I'm not keen about using commas here.  It's visually difficult to scan.
![image](https://github.com/user-attachments/assets/2a59698a-2b86-4bdf-a3cd-fc4f7e926ad2)

We've got a few different options here:

1. (easy) Increase the visual separation between the filenames.  This could be inserting more padding between the filenames or making the filenames be links to the repository (so they show up as a separate color to the separator).
2. (easy?) Separate the filenames with newlines -- not sure how nice this will play with the table layout.
3. Mimic the style of the Base Images section with each (Tool, Filename) be on its own row
  ![image](https://github.com/user-attachments/assets/c85aa15e-aed6-473f-b5c8-1a162afc3e55)

My preference would be 2, then 1, then 3, but we might want to get other opinions as well.

Reply:
How about as a list?

<img width="1241" alt="Screenshot 2024-12-19 at 12 03 27 PM" src="https://github.com/user-attachments/assets/ab43c916-f79b-4311-a335-204cb7b0d07e" />

- 2 as written didn't look right to me. The bullets make that a little better, IMO 
- 3 is unintuitive to me. The files are more justification for the assertion that the repository uses the given CI/CD Tool than a new finding themselves (like a new Tag is for the base images plugin)
- I agree that 1 with links to the files is ideal, but the CICD Tools plugin does not currently return a URL to the file. We could theoretically derive links from within the UI, but that'd be a lot of VCS-specific logic in the UI and it's unclear we'd have everything we need.

Reply:
Yeah, the bullets look great -- I think we've got a winner here.